test/persistence.test.js

```diff
@@ -94,6 +157,26 @@ describe('Persistence', function () {
     assert.deepStrictEqual(treatedData[1], { _id: '2', hello: 'world' })
   })
 
+  it('If two lines concern the same doc (= same _id), the last one is the good version (with stream)', function (done) {
+    const now = new Date()
+    const rawData = model.serialize({ _id: '1', a: 2, ages: [1, 5, 12] }) + '\n' +
+      model.serialize({ _id: '2', hello: 'world' }) + '\n' +
+      model.serialize({ _id: '1', nested: { today: now } })
+    const stream = new Readable()
+
+    stream.push(rawData)
+    stream.push(null)
+
+    d.persistence.treatRawStream(stream, function (err, result) {
+      var treatedData = result.data
```

Comment:
We should assert.isNull(err) and `var` should be replaced with `const` (standardjs)